transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/wizards/Hl7Page.java

```diff
@@ -117,7 +118,7 @@ public void handleValueChange(ValueChangeEvent event) {
                 }
                 if (path != null) {
                     try {
-                        IResource resource = model.getProject().findMember(path);
+                        IResource resource = CamelUtils.project().findMember(path);
```

Comment:
That was the entire point of this issue, to make the transformation editor only accessible via the Camel editor.  In fact, we had a couple of other issues logged that arose because we, some time after the new editor framework was introduced, changed code that assumed that already assumed an open and focused Camel editor.  Thus, things broke when the transformation editor was accessed directly via the new Fuse transformation action.  I was able to close those out as duplicates by removing that action in this fix and going with the idea that the Camel editor must be the only thing that launches the transformation editor.